lib/dialects/oracledb/index.js

```diff
@@ -342,6 +342,9 @@ Client_Oracledb.prototype._query = function(connection, obj) {
 function readStream(stream, type) {
   return new Promise((resolve, reject) => {
     let data = '';
+    if (type !== 'string') {
```

Comment:
why not only initialize data once, using ternary operator? (`let data = type === 'string'? '' : Buffer(0) `)

Reply:
I'm trying to follow the way it was before: https://github.com/knex/knex/commit/c9e305760d622071e1aaa6d8069cab25366d3b9b#diff-a86ace7e0baa34caa6f981181dc544e2

I'll see if ternary operators are common and implement if so.

Reply:
Ternary operators are pretty idiomatic in JS. And knex old code def not a good example of how to write code 😂

Reply:
Makes sense.  I'm a stickler for code style, but I get what you are saying.  :)

Reply:
Fixed!  Let me know if there's anything else!